packages/hash/frontend/src/components/sandbox/FramedBlock/FramedBlock.tsx

```diff
@@ -1,4 +1,4 @@
-import * as Sentry from "@sentry/nextjs";
+import * as Sentry from "@sentry/react";
```

Comment:
`FramedBlock.tsx` is bundled via this setting in `next.config.json`:

```
          new HtmlWebpackPlugin({
            filename: "static/sandbox.html",
            template: path.join(__dirname, framedBlockFolder, "index.html"),
            chunks: ["sandbox"],
          }),
```

This is a non-conventional way of bundling Next.js. Looks like a separate entry point affects webpack features in a custom bundle and it’s impossible to use `process.env.SOMETHING` in it. Replacing `@sentry/nextjs` with `@sentry/react` (which is a sub-dependency of `@sentry/nextjs`) resolve the issue. It also avoids similar errors like `unable to import "fs"`. This may be to do with the less advanced tree-shaking in a custom bundle.